pkg/service/update.go

```diff
@@ -18,6 +18,13 @@ type UpdateCommand struct {
 	manifest    manifest.Data
 	getInput    fastly.GetServiceInput
 	updateInput fastly.UpdateServiceInput
+
+	// TODO(integralist):
+	// ensure consistency in capitalization
+	// should be lowercase to avoid ambiguity in common.Command interface
```

Comment:
```suggestion
	// TODO(integralist):
	// Ensure consistency in capitalization, should be lowercase to avoid 
	// ambiguity in common.Command interface.
```